autoload/neomake/makers/ft/erlang.vim

```diff
@@ -10,3 +10,39 @@ function! neomake#makers#ft#erlang#erlc() abort
             \ '%E%f:%l: %m'
         \ }
 endfunction
+
+function! neomake#makers#ft#erlang#erlc_glob_paths() abort
+    return {
+        \ 'exe': 'erlc',
+        \ 'args': function('neomake#makers#ft#erlang#glob_paths'),
+        \ 'errorformat':
+            \ '%W%f:%l: Warning: %m,' .
+            \ '%E%f:%l: %m'
+        \ }
+endfunction
+
+function! neomake#makers#ft#erlang#glob_paths() abort
+    if match(expand('%'), 'SUITE.erl$') > -1
+        let l:profile = 'test'
+    else
+        let l:profile = 'default'
+    endif
+    let l:ebins = glob('_build/' . l:profile . '/lib/*/ebin', '', 1)
+    " Set g:erlang_extra_deps in a project-local .vimrc, e.g.:
+    "   let g:erlang_extra_deps = ['deps.local']
+    if exists('g:erlang_extra_deps')
+        for extra_deps in g:erlang_extra_deps
+            let l:ebins += glob(extra_deps . '/*/ebin', '', 1)
+        endfor
+    endif
```

Comment:
Is `g:erlang_extra_deps` something used by other plugins already?

Reply:
AFAIK it's not, I don't know it from my experience and I couldn't google up any suggestion it might be.
I added it because it's a quick way to support Rebar 2 projects or custom project structure in a reasonable way.

Reply:
Ok.  Then the variable name should include `neomake_`.
I suggest `neomake_erlang_erlc_extra_deps`.
It should also check a buffer local var.
I.e. `get(b:, 'neomake_erlang_erlc_extra_deps', get(g:, 'neomake_erlang_erlc_extra_deps'))`.